source/_docs/disaster-recovery.md

```diff
@@ -28,8 +36,19 @@ In the event of a zone failure we reroute your site to the backup zone and provi
 ### Real-time Failover Support
 Pantheon’s disaster response team will proactively notify you of a zone failure event and its impact to your site. Your dedicated team will keep clear and transparent lines of communication on any action taken to keep your site online, and is available 24/7 for all of your questions.
 
+## Redis & Disaster Recovery Best Practices
+Redis cache is not preserved after a site failover. This means you must ensure that the site can handle having its cache dropped under regular visitor traffic in order for the site to operate as expected in a failover scenario.
+
+To test in a Test Environment, click the **Clear Caches** button in the upper right hand corner of the Site Dashboard.
+You can also [connect to your Redis instance](/docs/redis/#use-the-redis-command-line-client) and run the `flushall` command to clear Redis cache via the Redis connection string.
+
+If you rely on the Redis cache for locks (mutexes) or storing other long-term data, you must move them out of Redis and into the database to avoid any issues when the Redis cache is dropped during failover.
+
 ## Limitations
-Redis and Solr are not currently supported on DR-enabled sites. See our documentation on how to [safely remove Redis](/docs/redis/#safely-remove-redis) and how to safely remove Solr for [Drupal 8](/docs/solr-drupal-8/#safely-remove-solr), [Drupal 7](/docs/solr-drupal-7/#safely-remove-solr), and [WordPress](/docs/wordpress-solr/#safely-remove-solr).
+Solr is not currently supported on DR-enabled sites. See our documentation on how to safely remove Solr for [Drupal 8](/docs/solr-drupal-8/#safely-remove-solr), [Drupal 7](/docs/solr-drupal-7/#safely-remove-solr), and [WordPress](/docs/wordpress-solr/#safely-remove-solr).
 
 ## See also
 - [Disaster Recovery Webinar - 11/8/2018 ](https://pantheon.io/resources/disaster-recovery-webinar){.external}
+
+## ChangeLog
```

Comment:
I thought you wanted to cut this?

Reply:
You who? I don't care one way or another.